packages/cli/src/api/upload.js

```diff
@@ -31,6 +32,7 @@ async function uploadPhrases(payload, params) {
       headers: {
         Authorization: `Bearer ${params.token}:${params.secret}`,
         'Content-Type': 'application/json;charset=utf-8',
+        'X-NATIVE-SDK': `txjs/cli/${version}`,
```

Comment:
maybe `'X-Native-Sdk-Version'`?

Reply:
This is already deployed on CDS and is expecting that header. The initial though process was around a "User-Agent" type of header: <SDK/VERSION>, thus I left the "version" name out.